<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox_userStatusBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_message">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>471</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sendMessage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyslij</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_imageBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sendImage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyslij</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_textMessage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wiadomosc tekstowa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_imageMessage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wiadomosc obrazkowa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_userStatus">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Twoj status:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_reveiverAddress">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>160</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_receiver">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>140</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Odbiorca:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_activeReceiver">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>270</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setReceiver">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>200</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Zapisz</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_receiver_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>240</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Obecnie wpisany odbiorca:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_receiverStatusLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>310</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status odbiorcy:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_receiverStatus">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>340</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
